### DISCUSSION TITLE
unable setup the node

### DISCUSSION THREAD

**Discussion by riyanparag1809** [Cyfrin/foundry-full-course-cu#2164] [General]
has in previous times genesis block add
but now its has updated to checkpoints 
iam unable to setup node

**Comment by EngrPips** [comment]
Hello @riyanparag1809, It is tough for me to comprehend your issue. Please visit https://github.com/Cyfrin/foundry-full-course-cu/blob/main/how-to-ask-a-question.md to know how to ask questions, as that will help people to help you properly.
